modules/src/core/ics04_channel/handler/send_packet.rs

```diff
@@ -59,9 +59,8 @@ pub fn send_packet(ctx: &dyn ChannelReader, packet: Packet) -> HandlerResult<Pac
 
     // check if packet height is newer than the height of the latest client state on the receiving chain
     let latest_height = client_state.latest_height();
-    let packet_height = packet.timeout_height;
 
-    if !packet.timeout_height.is_zero() && packet_height <= latest_height {
+    if !packet.timeout_height.is_zero() && packet.timeout_height <= latest_height {
```

Comment:
This change makes sense.

On a related topic, there is a comment above (L60) that is confusing me, specifically:

>    // check if packet height is newer than the height of the latest client state on the receiving chain

The receiving chain is the local chain (the one hosting & running the current ics04 module). It's not obvious to me what is the connection we meant to make between "latest client state" and "packet height". We should clarify that, or remove the comment if stale.

Reply:
It doesn't make sense to me either, so I'll just remove it.

Reply:
Removed a few comments that were either confusing or didn't say much [f8525a6867](https://github.com/informalsystems/ibc-rs/pull/2043/commits/f8525a6867c72f17e5d639363f2af0f6e6382fd9)